pkgs/by-name/sn/snakemake/package.nix

```diff
@@ -69,7 +73,9 @@ python3.pkgs.buildPythonApplication rec {
     snakemake-executor-plugin-cluster-generic
     snakemake-storage-plugin-fs
     stress
+    versionCheckHook
```

Comment:
have we established that versionCheckHook is okay to put in nativeCheckInputs rather than nativeInstallCheckInputs? I know the python builder uses installCheckPhase instead of checkPhase, but that is an implementation detail that could be changed in the future.

Reply:
Usually, `versionCheckHook` is added to `nativeInstallCheckInputs` indeed.
However, when I do that with python packages, the hook never runs. I have found that in Python packages, people would put this hook in `nativeCheckInputs` instead, so I have done the same (and it works).

Reply:
sounds like a future treewide :+1: